content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,90 @@ output:
 
 ## Introduction
 
-This tutorial will explain how you can match a list of 'scientific names' to the GBIF taxonomic backbone
+This tutorial will explain how you can match a list of _scientific names_ against the [GBIF backbone taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c).
 
-Important is that you have [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
+Important is that you have the most recent version of [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
 
-```{r message=FALSE, warning=FALSE}
-library(tidyverse) # tidyverse
-library(rgbif)     # To Match GBIF
-library(inborutils) # wrap GBIF api data
+```{r install_pkgs, eval=FALSE}
+devtools::install_github("ropensci/rgbif")    # install rgbif
```

Comment:
do you need to development version of rgbif?

Reply:
You are right. Indeed, not needed. Solved here: https://github.com/inbo/tutorials/pull/146/commits/c09cf9bd75a75f10f255441929348710fa153ce0